inst/pages/Introductory-workflow.qmd

```diff
@@ -0,0 +1,353 @@
+# Introductory
+
+<div id="workflow">
+
+<img src="images/logo.svg" alt="logo workflow" id="banner"/>
+
+# Introduction
+
+Hello and welcome to a complete workflow using the latest R/Bioconductor tools for microbiome data science. In this tutorial we’ll walk you through some basic steps of a composition analysis study using OMA. These will be applicable to almost any of your projects and will help you understand the fundamental concepts that will sky-rocket 🚀 your future microbiome analyses.
+
+# Importing data
+
+When using microbiome packages, there are many different ways to import your data. Let's first load the mia package:
+
+```{r}
+#| label: loadMia
+#| output: false
+library(mia)
+```
+
+```{r}
+#| label: loadDT
+#| echo: false 
+#| output: false
+library(DT)
+```
+
+You can either choose to use your own data or one of the built-in datasets provided by mia which you may find here @sec-example-data:
+
+In this tutorial, we'll be using the @Tengeler2020 dataset. This dataset was made by A.C. Tengeler to try to demonstrate the impact of altered microbiomes on the brain structure. Here's how we can load the data into our R environment:
+
+```{r}
+#| label: loadDataset
+data("Tengeler2020", package="mia")
+tse <- Tengeler2020
+```
+
+There are of course other ways to import your data using the [**mia package**](https://microbiome.github.io/mia/). These include: using either **your own data** @sec-import-from-file or by converting an existing object to a *TreeSummarizedExperiment* object as stated in @sec-conversions-between-data-formats-in-r.
+
+<hr>
+
+# Storing microbiome data
+
+*TreeSummarizedExperiment* or TreeSE object is the type of object used within the [**mia package**](https://microbiome.github.io/mia/) to store your data. It's a versatile and multi-purpose data type that allows for an efficient way to store and access data.
+
+Here's a quick reminder on how to access certain types of data:
+
+You can access [**assays**](https://microbiome.github.io/OMA/docs/devel/pages/04_containers.html#sec-assay-slot) as such:
+
+```{r}
+#| label: showAssay
+assay(tse)[1:5,1:10] 
+```
+
+Whereas [**colData**](https://microbiome.github.io/OMA/docs/devel/pages/10_manipulation.html#add-or-modify-data) is accessible through:
+
+```{r}
+#| label: showColdata
+# Transform the colData to a dataframe
+tse_df_colData <- as.data.frame(colData(tse))
+
+# Show as an interactive table
+datatable(tse_df_colData,options = list(pageLength = 5),rownames = FALSE)
+```
+
+[**rowData**](https://microbiome.github.io/OMA/docs/devel/pages/04_containers.html#rowdata) contains data on sample characteristics, particularly taxonomic information.
+
+```{r}
+#| label: setWarning
+#| echo: false 
+#| output: false
+options(warn = -1)
+```
```

Comment:
Instead, use warning=FALSE in a chunk where you want to silence warnings https://quarto.org/docs/computations/execution-options.html